### DISCUSSION TITLE
Cannot use Copilot Labs extension on Visual Studio Code: NoCopilotLabsAccess error

### DISCUSSION THREAD

**Discussion by sommoMicc** [community/community#75697] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I cannot find the way to sign up to Copilot Labs. Every time the extension loads, it gives me the error 'NoCopilotLabsAccess'.

I cannot find the way to enroll to Copilot Labs

**Comment by GabrielVeres95** [comment]
Having the same issue..

**Comment by x-bruce** [comment]
Having the same issue +1

**Comment by mackswift** [comment]
Same thing here

**Comment by adhershmnair** [comment]
Same here +1

![image](https://github.com/community/community/assets/91515825/7e68c6d1-21b6-4a46-95bb-e354dd3d2666)

![image](https://github.com/community/community/assets/91515825/c531ccd5-6fbc-4995-9896-947dee6893c9)

**Reply by DxBang** [reply]
Uninstall the Copilot preview edition, that should fix it. Just make sure Github Copilot and|or Github Copilot Chat is installed.

**Comment by MajdT51** [comment]
same

**Comment by fredy0219** [comment]
Same

**Comment by cohlar** [comment]
I had the same issue, just had to uninstall the `GitHub Copilot Labs` app from VS Code.
The tricky part was it didn't come up when searching for "GitHub Copilot Labs" so I couldn't find it (I assume it's because this app is retired)... but I found out that it's still listed as part of the Installed apps, just had to scroll and find it manually.

![image](https://github.com/community/community/assets/57757008/f1c78471-9ec4-4570-a2e8-604f7828ddba)

**Reply by Ayarmd** [reply]
how?

**Reply by Ayarmd** [reply]
i want istall github copilot labs in vsc i cant find

**Comment by amastaneh** [comment]
+1

**Comment by davecheney** [comment, marked as answer]
The Copilot Labs extension has been deprecated as the functionality has rolled into the Copilot and Copilot Chat extensions. Please uninstall the Copilot Labs extension.

**Reply by Oguidan** [reply]
But it still doesn't work and why.
